### DISCUSSION TITLE
init_worker_by_lua error: ./kong/global.lua:211: module 'resty.events.compat' not found:No LuaRocks module found for resty.events.compat

### DISCUSSION THREAD

**Discussion by liupenghui-owen** [Kong/kong#10509] [Help] [answered]
I start kong wit prefix "usr/local/kong".

2023/03/17 12:07:22 [error] 35505#0: init_worker_by_lua error: ./kong/global.lua:211: module 'resty.events.compat' not found:No LuaRocks module found for resty.events.compat

Actually, I found the file in "/bazel-bin/build/kong-dev/openresty/site/lualib/resty/events/compat/init.lua".

How to fix it? Thanks

**Comment by liupenghui-owen** [comment, marked as answer]
It works using below settings.
nginx_user = root root
nginx_main_user = root root

**Comment by ghost** [comment]
same as: #10508?
